registry/tenets.yaml

```diff
@@ -64,6 +64,8 @@ codelingo:
         hasIssues: false
       comment-first-word-as-subject:
         hasIssues: false
+      update-comment-first-word-as-subject:
```

Comment:
Interesting name - is that just because that's this is the updated version, or because this tenet is intended to do updating/rewriting? I reckon we should have a more descriptive name in either case!

Reply:
To suggest that it updates outdated comments! Hmm I gave it a few minutes thought but this is the best I could think of

Reply:
Great! Naming is worth a couple of minutes, but not worth getting stuck on. It's clear enough.